packages/expo-dev-menu/ios/Modules/DevMenuInternalModule.swift

```diff
@@ -202,4 +202,36 @@ public class DevMenuInternalModule: NSObject, RCTBridgeModule {
 
     resolve(DevMenuInternalModule.defaultScheme)
   }
+  
+  @objc
+  func getBuildInfoAsync(_ resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+    let bridge = manager.appInstance.bridge
+    
+    // TODO - pass manifest values to this method
+    let buildInfo = EXDevMenuBuildInfo.getFor(bridge as! RCTBridge, andManifest: [:] as Any)
+        
+    resolve([
+      "appName": buildInfo["appName"],
+      "appIcon": buildInfo["appIcon"],
+      "appVersion": buildInfo["appVersion"],
+      "runtimeVersion": buildInfo["runtimeVersion"],
+      "sdkVersion": buildInfo["sdkVersion"],
+      "hostUrl": buildInfo["hostUrl"],
+    ])
+  }
+  
+  @objc
+  func getDevSettingsAsync(_ resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+    let bridge = manager.delegate?.appBridge?(forDevMenuManager: self.manager)
```

Comment:
for some reason this bridge and the bridge in the previous method aren't the same? I needed to do this one to get the right devsetting values 🤷

Reply:
Actually, you should always use `manager.delegate?.appBridge` not `manager.appInstance.bridge`.

The second one (`manager.appInstance.bridge`) refers to the dev menu bridge. 
The first one (`manager.delegate?.appBridge`) refers to the currently active app bridge - so in practice, it is a dev launcher or app bridge.